skfda/exploratory/visualization/_ddplot.py

```diff
@@ -0,0 +1,139 @@
+    ) -> Figure:
+        """
+        Plot DDPlot graph.
+
+        Plot the depth of our fdata elements in the two different
+        distributions,one in each axis. It is useful to understand how
+        our data is more related with one subset of data / distribution
+        than another one.
+        Args:
+            chart (figure object, axe or list of axes, optional): figure over
+                with the graphs are plotted or axis over where the graphs are
+                plotted. If None and ax is also None, the figure is
+                initialized.
+            fig (figure object, optional): figure over with the graphs are
+                plotted in case ax is not specified. If None and ax is also
+                None, the figure is initialized.
+            axes (list of axis objects, optional): axis where the graphs
+                are plotted. If None, see param fig.
+            n_rows (int, optional): designates the number of rows of the figure
+                to plot the different dimensions of the image. Only specified
+                if fig and ax are None.
+            n_cols(int, optional): designates the number of columns of the
+                figure to plot the different dimensions of the image. Only
+                specified if fig and ax are None.
+            kwargs: if dim_domain is 1, keyword arguments to be passed to the
+                matplotlib.pyplot.plot function; if dim_domain is 2, keyword
+                arguments to be passed to the matplotlib.pyplot.plot_surface
+                function.
+        Returns:
+            fig (figure object): figure object in which the depths will be
+            scattered.
+        """
+        margin = 0.025
+        width_aux_line = 0.35
+        color_aux_line = "gray"
+
+        fig, axes = _get_figure_and_axes(chart, fig, axes)
+        fig, axes = _set_figure_layout_for_fdata(
+            self.fdata, fig, axes, n_rows, n_cols,
+        )
+
+        depth_dist1 = self.depth_method(
+            self.fdata, distribution=self.dist1,
+        )
+        depth_dist2 = self.depth_method(
+            self.fdata, distribution=self.dist2,
+        )
+
+        if self.fdata.dim_domain == 1:
+            for i in range(self.fdata.dim_codomain):
+                axes[i].scatter(
+                    depth_dist1,
+                    depth_dist2,
+                    **kwargs,
+                )
+
+        # Set labels of graph
+        fig.suptitle("DDPlot")
+        for axe in axes:
+            axe.set_xlabel("X depth")
+            axe.set_ylabel("Y depth")
+            axe.set_xlim(
+                [
+                    self.depth_method.min - margin,
+                    self.depth_method.max + margin,
+                ],
+            )
+            axe.set_ylim(
+                [
+                    self.depth_method.min - margin,
+                    self.depth_method.max + margin,
+                ],
+            )
+            axe.plot(
```

Comment:
What is this?

Reply:
i added a margin so  the points that are in the limit dont get cut

Reply:
I think that this can be done without adding lines to the Figure.

Reply:
oh, okay, no, the plot is a line used as a line that goes from 0 to 1 which corresponds to the line the points will tend to if both distributions are similar